src/App.jsx

```diff
@@ -42,16 +71,40 @@ function App() {
   setCards(newCardsList);
   }
 
-  console.log(cards);
-  
-  function checkDoneCards () {
-    const doneCards = cards.filter((card) => card.checked ==true);
-    console.log(`You have ${doneCards.length} tasks done so far`); // Shows but dont know how to concatenate in console.log
-  }  
+  // console.log(cards);
 
+
+  useEffect(() => {
```

Comment:
optional: its recommended to avoid using useEffect when dealing with internal state updates. This is a good read: https://react.dev/learn/you-might-not-need-an-effect